wso2.org.component.javaparser/src/main/java/component/javaparser/WSO2POMReader.java

```diff
@@ -0,0 +1,291 @@
+package component.javaparser;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+/**
+ * This class read and process the pom files of repository
+ */
+public class WSO2POMReader {
+
+    private void addSCRPluginVersion(Document doc, NodeList nodeList) {
+        Element ele = null;
+        String version = "1.16.0";
```

Comment:
Better if we can make this version an input parameter. At least make this a constant.. possible name: DEFAULT_SCR_PLUGIN_VERSION.

Reply:
fixed this one.